sdk/core/Azure.Core/src/ProcessingContext.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+
+namespace Azure.Core
+{
+    /// <summary>
+    /// Contains information related to the processing of the <see cref="HttpMessage"/>.
```

Comment:
It might be worth mentioning that processing means what happens when we send the message down and back the pipeline.

Reply:
Let me know if you are okay with the update.